.github/workflows/deploy-github-main.yml

```diff
@@ -4,6 +4,11 @@ on:
     workflows: ["Package"]
     types:
       - completed
```

Comment:
Can you also add the following below `types: - completed` so only trigger E2E builds after successful package builds:
```
conclusions:
      - success
```

Reply:
i somehow though this was explicit when using `completed`. Ive just added it, good catch!